api/models/users.models.js

```diff
@@ -1,18 +1,26 @@
 const db = require("../../db/connection.js");
 const format = require("pg-format");
 
-function validateUsername(username) {
+function selectUsers(username) {
```

Comment:
Your model takes a parameter here but you aren't invoking it with one in the controller. You don't need the username logic for this one. This endpoint should just return a list of users so you can delete the unused logic below 🙂